packages/context-editor/src/components/MenuBar.tsx

```diff
@@ -199,10 +242,10 @@ const Separator = () => {
 	);
 };
 
-export const MenuBar = () => {
+export const MenuBar = ({ upload }: { upload: Upload }) => {
 	return (
 		<div
-			className="flex items-center rounded-t border bg-gray-50 p-4"
+			className="flex items-center overflow-x-auto rounded-t border bg-gray-50 p-4"
```

Comment:
our menu bar is getting longer! adds horizontal scrolling to it

https://github.com/user-attachments/assets/3d34cc47-839e-44c4-8106-8d225400cae7